config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1812,6 +1812,15 @@
  #DEFAULT: none (null)
  # basiclti.provider.email.trusted.consumers=
 
+#Creates a site of mentioned type 
+#e.g., basiclti.provider.newsitetype=project
+#basiclti.provider.newsitetype=
+#default role mapping for the sakai provider site when could not find  matching role from consumer 
+#e.g., lti.role.mapping.Student=Member
+#lti.role.mapping.Student=
+#e.g.,lti.role.mapping.Instructor=Owner
+#lti.role.mapping.Instructor=
+
```

Comment:
Thanks for adding this! I know these were pre-existing properties, so this should help surface this feature